pyproject.toml

```diff
@@ -54,7 +54,7 @@ requires = [
     "setuptools>=61.2",
     "setuptools_scm[toml]>=3.6",
     "numpy>=1.25",
-#    "numpy>=2.0.0dev0",
+    #"numpy>=2.0.0dev0",
```

Comment:
I think this can be deleted altogether?
```suggestion
```

Reply:
Well, I left it there for easy switching to numpy 2.0 - that's the reason. Are you asking @jhunkeler ? I hope he has a better solution.

Reply:
Oh... this is _build_ requirements. In that case, I think you can just use numpy 2.x like astropy though you might want to pick up RC2 instead of RC1 by now.

https://github.com/astropy/astropy/blob/566fcbd4a36b787564a46b01c4c8322448076d99/pyproject.toml#L133

They claim the 2.x ABI is backward compatible for `numpy>=1.23`.

Reply:
but if I mention @jhunkeler a couple of times more, we might get an even better solution

Reply:
:zombie_man: 
If the code we have now is backward compatible and our pin is `numpy>=1.25`, then I suspect it won't be a big deal whenever 2.0 is released. The pins as they are right now (here^) will let tox do what it needs to do. We'll be able to test numpy 1.x and 2.x without any trouble from the build system.